### ISSUE TITLE
WARNING: No pipeline activities exist for org/proj/pr-1000

### ISSUE THREAD

**Issue by rishka** [jenkins-x-plugins/jx-slack#33]
I just installed the slack bot on a jx3 cluster, but every message is giving the above warning. It is updating messages in the channel, but no review request messages are showing up. 

I saw #13 & #14 but as far as I can tell, the labels are there for the pipeline resources

Can be reached in the jenkins-x-user slack at adam_p

**Comment by jstrachan** [comment]
what version of the slack chart is installed?

**Comment by rishka** [comment]
Slack chart version: 0.0.60(deployed today)
lighthouse-webhooks: gcr.io/jenkinsxio/lighthouse-webhooks:1.0.38

**Comment by rishka** [comment]
```
Preparing review request message for quintevents-qng-pr-4646-2
WARNING: No pipeline activities exist for quintevents/qng/pr-4646
```

```
  labels:
    build: "2"
    created-by-lighthouse: "true"
    event-GUID: f75fc570-ada5-11eb-9b3e-6a4a77e28c98
    lighthouse.jenkins-x.io/baseSHA: 00e568a5006cbc3f5b23f05315dba65451e0a324
    lighthouse.jenkins-x.io/branch: PR-4646
    lighthouse.jenkins-x.io/buildNum: "1620221391086"
    lighthouse.jenkins-x.io/context: pr
    lighthouse.jenkins-x.io/id: f8359fc9-ada5-11eb-8813-7af9e5f74f37
    lighthouse.jenkins-x.io/job: pr
    lighthouse.jenkins-x.io/lastCommitSHA: f1f27ad0ed56f036943681aa647116963dcb1e3e
    lighthouse.jenkins-x.io/refs.org: quintevents
    lighthouse.jenkins-x.io/refs.pull: "4646"
    lighthouse.jenkins-x.io/refs.repo: qng
    lighthouse.jenkins-x.io/type: presubmit
    podName: pr-x2bv8-from-build-pack-pzvg4-pod-jnjsz
    tekton.dev/pipeline: pr-x2bv8
```

**Comment by rishka** [comment]
As far as I can tell, it was searching for the wrong labels, but updating it to search for the "right" ones doesn't seem to make the slack messages work correctly either. 

https://github.com/jenkins-x-plugins/jx-slack/pull/34 got rid of the error messages in logs, but still no updates to the messages

**Comment by rishka** [comment]
This is fixed by #35
